ner_v2/detectors/numeral/number/standard_number_detector.py

```diff
@@ -21,18 +24,21 @@ def __init__(self, entity_name, data_directory_path, min_digit, max_digit):
         self.original_date_text = []
         self.entity_name = entity_name
         self.tag = '__' + entity_name + '__'
-        self.min_digit = min_digit
-        self.max_digit = max_digit
 
         self.numbers_word = {}
-        self.language_number_map = {}
+        self.language_scale_map = {}
+        self.units_map = {}
 
         # Method to initialise value in regex
         self.init_regex_and_parser(data_directory_path)
 
+        self.regex_numeric_patterns = re.compile(r'(([\d,]+[.]?[\d]*)\s?(' + "|".join(self.language_scale_map.keys())
+                                                 + r')?)\s*', re.UNICODE)
+
         # Variable to define default order in which detector will work
-        self.detector_preferences = [self._detect_number_from_numerals,
-                                     self._detect_numeric_digit]
+        self.detector_preferences = [self._detect_number_from_digit,
+                                     self._detect_number_from_numerals
```

Comment:
Given this order of detectors and the way _detect_number_from_digit works
something like `one thousand two hundred` would return `[1000, 200]` because <digit><scale> combo captures two things

Reply:
No, `_detect_number_from_digit` this always expect digit in front. It will not get detected from this method. 
for case `1 thousand 2 hundred` it will give [1000, 200].

Below is end to end result. 
```
In [3]: from ner_v2.detectors.numeral.number.number_detection import NumberDetector
In [4]: n = NumberDetector(entity_name='number', language='en')
In [6]: n.detect_entity('one thousand two hundred')
Out[6]: ([{'unit': None, 'value': '1200'}], ['one thousand two hundred'])

In [7]: n.detect_entity('1 thousand 2 hundred')
Out[7]: ([{'unit': None, 'value': '1000'}, {'unit': None, 'value': '200'}], ['1 thousand', '2 hundred']
```

I think these make sense